hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/ConfigurationProperties.java

```diff
@@ -158,39 +208,49 @@ private void copyProperties(
    */
   private void storePropertiesInPrefixNodes(Map<String, String> props) {
     for (Map.Entry<String, String> prop : props.entrySet()) {
-      List<String> propertyKeyParts = splitPropertyByDelimiter(prop.getKey());
-      if (!propertyKeyParts.isEmpty()) {
-        PrefixNode node = findOrCreatePrefixNode(nodes,
-            propertyKeyParts.iterator());
+      PrefixNode node = getNode(prop.getKey());
+      if (node != null) {
         node.getValues().put(prop.getKey(), prop.getValue());
-      } else {
-        LOG.warn("Empty configuration property, skipping...");
       }
     }
   }
 
+  /**
+   * Finds the node that matches the whole key or create it, if it does not exist.
+   * @param name name of the property
+   * @return the found or created node, if the name is empty, than return with null
+   */
+  private PrefixNode getNode(String name) {
+    List<String> propertyKeyParts = splitPropertyByDelimiter(name);
+    if (!propertyKeyParts.isEmpty()) {
+      return findOrCreatePrefixNode(null, propertyKeyParts.iterator());
+    } else {
+      LOG.warn("Empty configuration property");
```

Comment:
This is just noise like this. Add name to the log message or something more specific.